src/rf/js/src/home/components/processing-block.js

```diff
@@ -40,7 +40,7 @@ var LayerStatusComponent = React.createBackboneClass({
         }
 
         if (this.getModel().isDoneWorking()) {
-            actionLink = (<a onClick={this.dismiss}>Dismiss</a>);
+            actionLink = (<a href="#" onClick={this.dismiss}>Dismiss</a>);
```

Comment:
Thanks. Sorry I missed that.
Generally I'm in favor of fixing that in CSS rather than junking up the markup with links that don't go anywhere.
```css
a {
  cursor: pointer;
}
```